css-three-column-layout/styles.css

```diff
@@ -0,0 +1,43 @@
+.container {
+  max-width: 950px;
+  margin: 0 auto;
+  background-color: #1996da;
+  padding: 25px 50px;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  padding: 0 10px;
+}
+
+.column-one-third {
+  flex-basis: calc(100% / 3);
+  text-align: center;
+  align-items: center;
+}
+
+.circle {
+  width: 100px;
+  height: 100px;
+  background-color: #fff;
+  border-radius: 50%;
+  margin: 0 auto -70px;
+
+  /* Give the circle a margin top 0, auto for L/R to center it and -70px at the bottom to bring circle up */
+}
+
+.image {
+  max-width: 100%;
```

Comment:
adjust the width accordingly after making the circle a wrapper, so that you have room to center it within the wrapper.